assembl/static2/js/app/components/debate/multiColumns/postColumn.jsx

```diff
@@ -3,33 +3,12 @@ import React from 'react';
 
 import Tree from '../../common/tree';
 import ColumnHeader from './columnHeader';
-import ColumnsPost from '../../../components/debate/multiColumns/columnsPost';
 import FoldedPost from '../common/post/foldedPost';
-import BoxWithHyphen from '../../common/boxWithHyphen';
+import ColumnsPost from '../../../components/debate/multiColumns/columnsPost';
+import ColumnSynthesis, { type Props as ColumnSynthesisProps } from './columnSynthesis';
```

Comment:
why not name `Props` directly `ColumnSynthesisProps` in `columnSynthesis.jsx`?

Reply:
In the end, I prefer to use the shorter `Props` and `State` names next to the component (the short version is "light" and we rarely need to import types from other components).